certificate-manager/src/main/java/io/strimzi/certs/Cert.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2017-2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.certs;
+
+public class Cert {
```

Comment:
We have to give it the best name for the code as it is today, I think. It would be easy enough to change the class name at the time we added the extra thing. Or maybe if there was a need for an extra thing in the future that would be a different class (maybe a subclass).

Reply:
ok